src/Scientist/Internals/ConcurrentSet.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System.Collections.Concurrent;
+using System.Collections.Generic;
+
+namespace GitHub.Internals
+{
+	internal class ConcurrentSet<T>
+	{
+		static readonly byte One = 1;
```

Comment:
Minor nit, but this entire implementation looks indented too much. We indent 4 spaces.